helpers/helpers.v2.1.d/backup

```diff
@@ -45,7 +45,14 @@ ynh_backup() {
     local is_data=false
 
     # If the path starts with /var/log/$app or $data_dir
-    if [[ (-n "${app:-}" && "$target" =~ ^/var/log/$app/?\*?$) || (-n "${data_dir:-}" && "$target" =~ ^$data_dir)     ]]; then
+    if [[ \
+        # Apps backup
+        (-n "${app:-}" && "$target" =~ ^/var/log/$app/?\*?$) \
+        || (-n "${data_dir:-}" && "$target" =~ ^$data_dir) \
+        # for core backups
+        || (-z "${app:-}" && "$target" == '/home/'*) \
+        || (-z "${app:-}" && "$target" == '/var/mail') \
```

Comment:
It's unclear why this was changed, this may change some existing behavior such as people calling something along the lines of:

```shell
BACKUP_CORE_ONLY=1 yunohost backup create
```

hoping to backup "everything" (apps and core) but without the app's core data ... with this change it would also exclude the /home/ and /var/mail dirs

(dunno how much of a big deal this is...)

Reply:
Update: supposedly that's because the core called `ynh_backup foo 1` with the "1" corresponding to the legacy "is_big" uuueargha